components/Button/index.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+const StyledButton = styled.button`
+    width: 10rem;
+    height: 2rem;
+    background-color: #0a0a23;
+    color: #fff;
+    border: none;
+    border-radius: 10px;
+    box-shadow: 0px 0px 2px 2px rgb(0, 0, 0);
+    
+    
+  `;
+  export default function Button({ type, children, onClick}) {
+  
+
+  return <StyledButton type={type} onClick={onClick}>{children}</StyledButton>;
+}
```

Comment:
Looks like prettier is not working on saving a file. Please check the VSCode config for the prettier extension.